core/src/global.css

```diff
@@ -25,6 +25,12 @@ html:not(.x) {
 	--support-3: hsl(221, 100%, 61%);
 	--support-4: hsl(217, 85%, 49%);
 
+	--success-2: hsl(118, 57%, 92%);
+	--success-5: hsl(140, 63%, 41%);
+
+	--failure-2: hsla(0, 76%, 94%, 1);
+	--failure-5: hsla(0, 100%, 34%, 1);
+
```

Comment:
Hm.. only updating some values in a scale is not a good idea. The other shades (e.g. success-1, success-3) would not match with this.

We should do either:
1. Update the whole scale to follow Banhmi's design, or
2. Just use the current scale of Moai success and failure